api/client/client.go

```diff
@@ -1137,6 +1147,16 @@ func (c *Client) GetMFADevices(ctx context.Context, in *proto.GetMFADevicesReque
 	return resp, nil
 }
 
+// GetMFAAuthenticateChallengeWithAuth gets mfa challenges for the currently logged in user.
```

Comment:
```suggestion
// GetMFAAuthenticateChallengeWithAuth gets MFA challenges for the currently logged in user.
```

MFA is an acronym, so let's capitalize it in the godocs.

Reply:
+1